config/config.rb

```diff
@@ -26,22 +26,22 @@ module Config
 
   # Override -- value is returned or the set default.
   override :database_timeout,       10,               int
-  override :db_pool,                5,                int
+  override :db_pool_max_size,       3,                int
```

Comment:
We already use 3 on production for a long time, because 5 is too many – this value is per process and Puma spawns multiple processes.